client/src/components/_common/UserSearchbar/UserSearchbar.jsx

```diff
@@ -60,13 +60,13 @@ const UserSearchbar = ({
       <div className={`input-group ${styles['member-search-group']}`}>
         <div className="input-group-prepend">
           <Button
-            className={`${styles['add-button']} ${styles['member-search']}`}
+            type="primary"
```

Comment:
The [design](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/82d25240-6e20-491e-9f90-f7c9d534965a/specs/) shows a gray button, which is what would be a white secondary button in the [new button styles](https://xd.adobe.com/view/b37b291a-7798-4814-90f3-7ac63c48a3e0-e628/screen/d7a72466-4ae8-43fd-9531-44e033245ddb/). The _white_ button and _white_ input lack distinction form each other. [I've asked design whether this is a concern.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1653600337362999)

Reply:
[I got a designer vote for purple.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1653600470309609) That's enough for me. I'll update the UI patterns note to account for the use case.

Reply:
This should be `type="secondary"` __after__ [FP-1677](https://jira.tacc.utexas.edu/browse/FP-1677) (which I am working on now).